cirq-web/cirq_ts/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "./node_modules/gts/",
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "rules": {
+        "node/no-unpublished-import": "off",
+        "max-len": [2, 120]
```

Comment:
can we make this 100? It is 100 in python.

Reply:
Sounds good.